lib/relay.js

```diff
@@ -396,3 +400,10 @@ function logError(logContext, error) {
     'error while sending websocket request over HTTP connection',
   );
 }
+
+function sanitizeHeaders(headers) {
```

Comment:
have you seen the existing `sanitiseHeaders` function? Probably should do it there so that it applies everywhere

Reply:
No I haven't. Great call, will amend it!